geoviews/plotting/bokeh/__init__.py

```diff
@@ -310,5 +310,6 @@ def _process(self, element, key=None):
 options.Feature.Ocean  = Options('style', fill_color='#97b6e1', line_color='#97b6e1')
 options.Feature.Lakes  = Options('style', fill_color='#97b6e1', line_color='#97b6e1')
 options.Feature.Rivers = Options('style', line_color='#97b6e1')
+options.Feature.States = Options('style', fill_alpha=0)
```

Comment:
I guess Borders is a polyline rather than a polygon, and thus unfilled?  Looks like Land is filled by default, but I guess that makes more sense. So I guess this is the one change that's needed!